### PR TITLE
127 search expander blue page

### PR DESCRIPTION
**Pull request by Kestutisjulius**

### CODE REVIEW

**Review by ArturasKi — APPROVED** [review]

**Review by LeoGytis — DISMISSED** [review]

**Review by LeoGytis — COMMENTED** [review]
Git conflict

File: css/pages/home.css [hunk 1]
```diff
@@ -14,6 +14,7 @@
 @import '../components/lets-talk.css';
 @import '../components/blogs.css';
 @import '../components/services.css';
+@import '../components/search.css';
```
Comment (Kestutisjulius) [inline css/pages/home.css:17]: prisidejo eilute

File: js/components/search/search.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+
+function search(selector) {
+        const DOM = document.querySelectorAll(selector);
+        selector.addEventListener('click', DOM.querySelector('.searchbox').style.left = "40%");
+        console.log(DOM);
+}
+export { search };
```
Comment (Kestutisjulius) [inline js/components/search/search.js:8]: sits kolkas neaktualus

File: js/pages/home.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ import { whyChooseUs } from '../data/whyChooseUs.js';
 import { renderIconList } from '../components/lets-talk/lets-talk.js';
 import { letstalkData } from '../data/letstalkData.js';
 import { renderBlogs } from '../components/blogs/blogs.js';
```
Comment (Kestutisjulius) [inline js/pages/home.js:11]: iterpiau eilute

File: js/pages/home.js [hunk 4]
```diff
@@ -23,8 +24,23 @@ addEventListener('scroll', function () {
...
-})
-
+});
+addEventListener('click', function(){
+       const searchDOM = document.querySelector('#search');
+       
+       if(searchDOM.click){
+           searchDOM.querySelector('.searchbox').style.left = "0%";
+        }
+    });
```
Comment (Kestutisjulius) [inline js/pages/home.js:34]: query selectorius prisidejo

File: js/pages/home.js [hunk 5]
```diff
@@ -23,8 +24,23 @@ addEventListener('scroll', function () {
...
+    
+/*     addEventListener('click', function(){
+        const searchDOM = document.querySelector('#search');
+        const searchX = document.querySelector('#x');
+        
+    if(searchX.click){
+     searchDOM.querySelector('.searchbox').style.left = "100%";
+    }
+}); */
```
Comment (Kestutisjulius) [inline js/pages/home.js:43]: palikau kaip komentara

**Review by ArturasKi — APPROVED** [review]

**Review by LeoGytis — APPROVED** [review]

**Review by ArturasKi — APPROVED** [review]

**Pull request merged**